css/style.css

```diff
@@ -1 +1,1461 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
+  --secondary-color: rgba(20, 27, 34, 1);
+  --article-bg-color: rgba(255, 255, 255, 0.05);
+  --main-background-color: rgba(13, 13, 17, 1);
+  --text-color: rgba(194, 195, 197, 1);
+  --inactive-color: rgba(136, 139, 142, 1);
+  --text-font: "Poppins", "Helvetica Neue", Arial, sans-serif;
+  --btn-header-font: "Bakbak One", "Arial Black", Arial, sans-serif;
+}
+
+/********************** General css **********************************/
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: rgba(4, 11, 17, 1);
+  font-family: "Bakbak One", "Arial Black", Arial, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 28px;
+  color: rgba(255, 255, 255, 1);
+}
+
+a {
+  text-decoration: none;
+  color: rgba(255, 255, 255, 1);
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  padding: 0 255px;
+}
+
+/**************** Buttons styles ****************/
+
+button {
+  cursor: pointer;
+}
+
+.basic-btn {
+  padding: 12px 24px;
+  font-size: 16px;
+  line-height: 24px;
+  gap: 8px;
+}
+
+.media-btn img {
+  width: 24px;
+  height: 24px;
+  background-color: transparent;
+}
+
+.btn-light {
+  border: none;
+  background-color: var(--btn-primary-color);
+  color: var(--secondary-color);
+  font-family: var(--btn-header-font);
+}
+
+.btn-border {
+  border: 2px solid var(--btn-primary-color);
+  background-color: inherit;
+  color: rgba(255, 255, 255, 1);
+  font-family: var(--btn-header-font);
+}
+
+.btn-border:active {
+  background-color: rgba(33, 231, 134, 0.2);
+  transition: all 0.3s ease-in-out;
+}
+
+/* Text styles */
+.highlighted-text {
+  color: var(--btn-primary-color);
+}
+
+.secondary-heading {
+  font-size: 44px;
+  line-height: 56px;
+  text-align: center;
+}
+
+/********************* Header styles *****************************/
+
+header {
+  margin: 30px 0;
+}
+
+.logo-block {
+  font-size: 32px;
+  line-height: 42px;
+  gap: 16px;
+}
+
+.logo-block img {
+  width: 36px;
+  height: 40px;
+}
+
+/************* Header navigation styles *******************/
+
+.nav-menu-items {
+  gap: 32px;
+}
+
+.nav-menu-item a {
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 10%;
+}
+
+.arrow-dwn {
+  width: 15px;
+  height: 15px;
+  margin-left: 5px;
+}
+
+.dropdown-content a {
```

Comment:
Using tag names for styling s a bad practise - better create class name for this link - for example `dropdown-content-link`

Reply:
#fixed